spec/index.bs

```diff
@@ -8719,6 +8723,13 @@ dictionary GPUFragmentState
                     struct and [=use dual source blending=].
             - [$Validating GPUFragmentState's color attachment bytes per sample$](|device|, |descriptor|.{{GPUFragmentState/targets}}) succeeds.
         </div>
+
+        <div class=compatmode>
```

Comment:
The class is just for styling, there must also be actual text that explains this (see `Stylistic labels` comment in the CSS).

I would suggest something like what I'm pushing to the PR. I also updated a few other things that needed to be removed.

Reply:
Note the box is inside the validusage box, since it still is part of the valid usage rules. It reads like any other validation that requires some optional feature, except it's highlighted by the compatmode box.